Articles/WhatIs/WhatIsInPersonLearning.md

```diff
@@ -0,0 +1,27 @@
+
+### What Is In-Person Learning?
+
+<!--- deck start --->
+In-person training provides invaluable opportunities for hands-on experience, direct feedback, and meaningful interaction that might be difficult to replicate in other settings.
+<!--- deck end --->
+
+<!--- body start --->
+In-person learning entails a training or development program conducted at a physical workplace or training center, where employees attend courses, workshops, or training sessions in person, rather than through online or remote means. This type of training provides employees with hands-on experience, interactive sessions, and face-to-face interaction with trainers and colleagues.
+
+In-person learning is important in scientific computing because it provides an opportunity for students to interact in a collaborative and engaging learning environment. Scientific computing involves the application of computational methods and tools to solve complex scientific problems, and in-person learning allows students to receive topic-specific instruction, understand pitfalls and receive practical guidance from experts in the field. Motivation and engagement are known to be higher and distraction is known to be lower during in-person learning, rather than online or self-study because it provides a structured, interactive and  a more accountable learning environment. This can lead to better learning outcomes and a deeper understanding of scientific field under study.
```

Comment:
```suggestion
In-person learning is important in scientific computing because it provides an opportunity for students to interact in a collaborative and engaging learning environment. Scientific computing involves the application of computational methods and tools to solve complex scientific problems, and in-person learning allows students to receive topic-specific instruction, understand pitfalls and receive practical guidance from experts in the field. Motivation and engagement are known to be higher, while distraction is known to be lower during in-person learning, rather than online or self-study because it provides a structured, interactive and  a more accountable learning environment. This can lead to better learning outcomes and a deeper understanding of the scientific field under study.
```